scripts/docker/alpine/3.10/release.Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM alpine:3.10
+
+RUN apk update \
+    && apk add --no-cache --upgrade\
+        bison \
+        build-base \
+        cmake \
+        curl \
+        libgit2 \
+        git \
+        ninja \
+        tcl \
```

Comment:
Are you sure we need [Tcl](https://en.wikipedia.org/wiki/Tcl) for Elektra? As far as I know no part of Elektra requires the programming language. If you want to include the [`tcl` plugin](https://www.libelektra.org/plugins/tcl), then I think adding a package for [Boost](https://www.boost.org) would be the right choice.

Reply:
Yes, the idea was to include the tcl plugin as it was part of our asciinema tutorial.

I saw tcl being installed in other docker images so I thought this is needed for the plugin. Thanks for reviewing it, I will add boost.

Further suggestions are absolutely welcome.

Reply:
Yes, this is a common error :wink: Please remove the references of the tcl dependency in the docker images. Elektra never used the tcl interpreter.